services/common/src/interfaces/dropDownListItem.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface IDropDownListItem {
```

Comment:
There is already an IOption interface here `services/common/src/interfaces/common/option.interface.ts`.  The only difference is that the value is `string`, but we can change it to `string | number` without any problem.

Reply:
`IOption` has been adjusted now and `IDropDownListItem ` removed